### PR TITLE
flash: Support --image arg to specify local chromiumos image file

### PR DESCRIPTION
**Pull request by hikalium**
TEST=lium flash --usb --image ${PATH_TO_IMAGE}

### CODE REVIEW

File: src/cmd/flash.rs [hunk 1]
```diff
@@ -55,62 +59,67 @@ pub fn run(args: &Args) -> Result<()> {
...
-                // The board name may have suffix '64' or '-*', so get the first alphabet
-                // sequence as the base board name.
-                let re = Regex::new(r"(^[[:alpha:]]*)")?;
-                let cap_arg = if let Some(cap) = re.captures(board_from_arg) {
-                    cap
+    let image_path = if let Some(image) = &args.image {
+        // If --image is specified, use the local file
+        image.clone()
+    } else {
+        // Determine a BOARD to flash
```
Comment (mhiramat) [inline src/cmd/flash.rs:66]: Can you make this `else` block a local function?

Comment (hikalium) [reply src/cmd/flash.rs]: The part is extracted into determine_board_to_flash(). Please review again!

Comment (HidenoriKobayashi) [reply src/cmd/flash.rs]: I'm curious if the original comment meant  the whole thing in the else block or the part that was extracted as `determine_board_to_flash`

Comment (hikalium) [reply src/cmd/flash.rs]: Not sure what do you mean... @mhiramat Could you checkif the change is acceptable and aligned with your suggestion? Thanks!!!

Comment (hikalium) [reply src/cmd/flash.rs]: ping...

**Comment by hikalium** [comment]
@mhiramat could you review this again? Thank you!

File: src/cmd/flash.rs (hunk 1 above)
Comment (mhiramat) [reply src/cmd/flash.rs]: Sorry for later. Yes, @HidenoriKobayashi is right. The else block seems independently works.

**Review by mhiramat — APPROVED** [review]
This looks good to me. Thanks!

**Pull request merged**